User profile

Carduus0 opened the pull request.

Carduus0 committed:
feat: add pageprofile.vue, add patch to router, add name and email

Carduus0 committed:
feat: add a user card with a link to the profile page

Carduus0 committed:
feat:add page-service-providers

Aleksstepanov commented on src/layouts/default/LayoutDefault.vue:
```diff
@@ -12,55 +11,93 @@
           <p class="q-mb-none">Сервис обработки платежных квитанций</p>
         </q-toolbar-title>
-        <q-btn icon="more_vert" flat/>
+        <q-btn icon="more_vert" flat @click="openBurger" />
```
Сделай <q-btn>
<q-menu
....
</q-btn>
Т.е. компонент меню пускай будет children для q-btn, как в доке у quasar

Aleksstepanov commented on src/layouts/default/LayoutDefault.vue:
```diff
@@ -12,55 +11,93 @@
+          <div class="row no-wrap q-pa-md">
+            <div class="column">
+              <div class="text-h6 q-mb-md">Settings</div>
+              <q-toggle v-model="mobileData" label="Use Mobile Data" />
```
Это надо удалить

Aleksstepanov commented on src/layouts/default/LayoutDefault.vue:
```diff
@@ -12,55 +11,93 @@
+            <div class="column">
+              <div class="text-h6 q-mb-md">Settings</div>
+              <q-toggle v-model="mobileData" label="Use Mobile Data" />
+              <q-toggle v-model="bluetooth" label="Bluetooth" />
```
Этот хардкод тоже удаляем

Aleksstepanov reviewed (commented):
review

Carduus0 committed:
feat: add btn component

Aleksstepanov committed:
fix:minorr
Squashed commit of the following:

commit dce1ff49e4ac2b74e0e1bffb36c9013185d55678
Author: Aleksey Stepanov <stepanovl87@gmail.com>
Date:   Thu Aug 3 18:16:30 2023 +0300

    add logout logik

commit 3b785212cc83ea4b5e5c62dc851c44daf605badd
Author: Aleksey Stepanov <stepanovl87@gmail.com>
Date:   Thu Aug 3 17:49:02 2023 +0300

    add components conformition dialog

Aleksstepanov committed:
fix:minorr
